charts/enterprise-metrics/values.yaml

```diff
@@ -1095,3 +1093,195 @@ minio:
       cpu: 100m
       memory: 128Mi
   secretKey: supersecret
+
+graphite:
+  # If true, enables graphite querier and graphite write proxy functionality.
+  # Read more in https://grafana.com/docs/metrics-enterprise/latest/graphite/
+  enabled: true
+
+graphite_querier:
+  replicas: 2
+
+  schemasConfiguration:
```

Comment:
Does it make sense to include some docs on how to set this up? Like links to our own docs or graphite docs.